app/connectors/AmlsNotificationConnector.scala

```diff
@@ -22,11 +22,10 @@ import models.notifications.{NotificationDetails, NotificationRow}
 import play.api.Logger
 import play.api.libs.json.Writes
 import uk.gov.hmrc.http._
-import uk.gov.hmrc.play.bootstrap.http.HttpClient
 import uk.gov.hmrc.play.http.ws.WSHttp
 
 import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Let's remove the global one and bring back the injected EC to keep it consistent